### ISSUE TITLE
md-button-toggle exclusive selection not working with mdModel

### ISSUE THREAD

**Issue by buu700** [angular/material2#4144]
#### Bug, feature request, or proposal:

From the documentation:

> By default, `md-button-toggle-group` acts like a radio-button group- only one item can be selected. In this mode, the `value` of the `md-button-toggle-group` will reflect the value of the selected button and `ngModel` is supported.

#### What is the expected behavior?

What the documentation says.

#### What is the current behavior?

`Error: No value accessor for form control with unspecified name attribute`

#### What are the steps to reproduce?

Run https://plnkr.co/edit/ohM2H6N4OuQNHECubeSU with console open.

#### Which versions of Angular, Material, OS, browsers are affected?

Tested with 2.0.0-beta.3.

**Comment by kara** [comment]
@tinayuangao IIRC, only button-toggle-group actually supports form directives, not button toggle alone. Can you confirm if button-toggle should work independently of button-toggle-group?  We should probably update the docs if not.

**Comment by buu700** [comment]
Oh. Oops, sorry about that. I saw "Exclusive selection vs. multiple selection" in the docs and just assumed that the former wouldn't need a group container and mentally skipped over every instance of `-group` in that section of the documentation I quoted.

(I'd mistakenly expected this to be a slight variation on md-slide-toggle, which had me a bit confused going into it, but after reading through the documentation more carefully I see that I'd misread a number of things on the first pass.)
